modulemd/modulemd-module-stream-v2.c

```diff
@@ -2856,6 +2987,10 @@ modulemd_module_stream_v2_emit_yaml (ModulemdModuleStreamV2 *self,
       return FALSE;
     }
 
+  if (modulemd_module_stream_v2_is_static_context (self))
+    {
+    }
```

Comment:
I'm only reviewing the last commit at the moment. Looks like the emit code didn't make it into this commit. Some specific tests for parsing/emitting static_context would be in order--which would have caught this. Everything else that's here looks good too me!

Reply:
Whoops! :anguished: Yeah, I think I got distracted and forgot to add the emit. Thanks for catching that.

I've added a test as well now.